pytripgui/kernel_vc/kernel_view.py

```diff
@@ -0,0 +1,225 @@
+    def comment(self):
+        return self.ui.comment_lineEdit.text()
+
+    @comment.getter
+    def comment(self):
+        return self.ui.comment_lineEdit.text()
+
+    @comment.setter
+    def comment(self, comment):
+        self.ui.comment_lineEdit.setText(comment)
+
+    @property
+    def projectile_name(self):
+        return self.ui.projectileName_lineEdit.text()
+
+    @projectile_name.getter
+    def projectile_name(self):
+        return self.ui.projectileName_lineEdit.text()
+
+    @projectile_name.setter
+    def projectile_name(self, projectile_name):
+        self.ui.projectileName_lineEdit.setText(projectile_name)
+
+    @property
+    def projectile_symbol(self):
+        return self.ui.symbol_comboBox.currentData()
+
+    @projectile_symbol.getter
+    def projectile_symbol(self):
+        return self.ui.symbol_comboBox.currentData()
+
+    @projectile_symbol.setter
+    def projectile_symbol(self, projectile_symbol):
+        index_of_symbol = self.ui.symbol_comboBox.findData(projectile_symbol, Qt.UserRole)
+        if index_of_symbol == -1:
+            raise Exception("Given symbol wasn't found on the list")
+        self.ui.symbol_comboBox.setCurrentIndex(index_of_symbol)
+
+    @property
+    def z(self):
+        return self.ui.z_spinBox.value()
+
+    @z.getter
+    def z(self):
+        return self.ui.z_spinBox.value()
+
+    @z.setter
+    def z(self, z):
+        self.ui.z_spinBox.setValue(z)
+
+    @property
+    def a(self):
+        return self.ui.a_spinBox.value()
+
+    @a.getter
+    def a(self):
+        return self.ui.a_spinBox.value()
+
+    @a.setter
+    def a(self, a):
+        self.ui.a_spinBox.setValue(a)
+
+    @property
+    def kernel_name(self):
+        return self.ui.kernelName_lineEdit.text()
+
+    @kernel_name.getter
+    def kernel_name(self):
+        return self.ui.kernelName_lineEdit.text()
+
+    @kernel_name.setter
+    def kernel_name(self, kernel_name):
+        return self.ui.kernelName_lineEdit.setText(kernel_name)
+
+    def add_kernel_with_name(self, kernel, kernel_name):
+        self.ui.beamKernel_comboBox.addItem(kernel_name, kernel)
+
+    def select_recently_added_kernel(self):
+        kernel_index = self.ui.beamKernel_comboBox.count() - 1
+        self.ui.beamKernel_comboBox.setCurrentIndex(kernel_index)
+
+    def remove_current_kernel(self):
+        kernel_index = self.ui.beamKernel_comboBox.currentIndex()
+        self.ui.beamKernel_comboBox.removeItem(kernel_index)
+
+    def get_all_kernels(self):
+        kernels_count = self.ui.beamKernel_comboBox.count()
+        kernels = list()
+        for i in range(kernels_count):
+            kernel = self.ui.beamKernel_comboBox.itemData(i, Qt.UserRole)
+            kernels.append(kernel)
+        return kernels
+
+    def replace_kernel_by_index(self, kernel, kernel_index):
+        self.ui.beamKernel_comboBox.setItemData(kernel_index, kernel, Qt.UserRole)
+
+    def setup_all_available_projectile_symbols(self, symbols):
```

Comment:
Dictionary of projectiles is not necessarily sorted:
```
>>> 
(pytripgui) grzanka@grzanka-VirtualBox:~/PycharmProjects/pytripgui$ PYTHONPATH=. python 
Python 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:41:56) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pytrip.tripexecuter import Projectile
>>> Projectile._projectile_defaults
{'H': (1, 1), 'O': (8, 16), 'Ar': (16, 40), 'He': (2, 4), 'Ne': (10, 20), 'Li': (3, 7), 'C': (6, 12)}
>>> for symbol in Projectile._projectile_defaults:
...    print(symbol)
... 
H
O
Ar
He
Ne
Li
C
```

Please add proper sorting here. An example of something ugly:

```
>>> [y[0] for y in sorted(Projectile._projectile_defaults.items(), key=lambda x : x[1][0])]
['H', 'He', 'Li', 'C', 'O', 'Ne', 'Ar']
```